pkgs/applications/networking/browsers/ladybird/default.nix

```diff
@@ -30,6 +31,18 @@ in gcc11Stdenv.mkDerivation {
     hash = "sha256-XQj2Bohk8F6dGCAManOmmDP5b/SqEeZXZbLDYPfvi2E=";
   };
 
+  postPatch = ''
+    mkdir serenity
+    cp -r ${serenity}/* serenity
+    chmod -R +w serenity
+  ''
```

Comment:
This should probably go in postUnpack, or we should use `srcs`

Reply:
Using `srcs` now.